numpy/lib/utils.py

```diff
@@ -496,7 +499,7 @@ def info(object=None, maxwidth=76, output=sys.stdout, toplevel='numpy'):
         Printing width.
     output : file like object, optional
         File like object that the output is written to, default is
-        ``stdout``.  The object has to be opened in 'w' or 'a' mode.
+        ``None``.  The object has to be opened in 'w' or 'a' mode.
```

Comment:
```suggestion
        ``None``, in which case ``sys.stdout`` will be used.
         The object has to be opened in 'w' or 'a' mode.
```

Reply:
One nit in the docstring

Reply:
Thank you for review. I am going to fix it.